README.md

```diff
@@ -456,50 +456,91 @@ The following settings are optional and allow you to further configure your clus
 
   See the [Roles Anywhere documentation](https://docs.aws.amazon.com/rolesanywhere/latest/userguide/credential-helper.html) for more details on the `aws_signing_helper` arguments.
 
+* `settings.kubernetes.disable-cpu-cfs-enforcement`: Whether to disable enforcing quotas for containers that specify CPU limits.
```

Comment:
I had a comment on this in the code, but I'm not seeing it now. I must have accidentally dropped it in a rebase. 

The problem that led me to make it `disable-*` is due to handlebars. Having `cpu-cfs-quota-enforced = false` is a problem since the default is that it is enforced. So needing to set it to false makes the templating a challenge with it being evaluated as "falsey".

I'll take another look, but I don't think this worked:

```
{{#if (eq settings.kubernetes.cpu-cfg-quota-enforced false)}}
enforced = false
{{/if}}
```

I'll take another shot and see if I can find a better way.

Reply:
Figured it out - brain got stuck on the wrong pattern. :]